documentation/assemblies/deploying/assembly-deploy-kafka-connect.adoc

```diff
@@ -8,23 +8,19 @@
 
 [role="_abstract"]
 link:https://kafka.apache.org/documentation/#connect[Kafka Connect^] is a tool for streaming data between Apache Kafka and external systems.
+Kafka Connect is typically used to integrate Kafka with external databases and storage and messaging systems.
 
 In Strimzi, Kafka Connect is deployed in distributed mode.
 Kafka Connect can also work in standalone mode, but this is not supported by Strimzi.
 
 Using the concept of _connectors_, Kafka Connect provides a framework for moving large amounts of data into and out of your Kafka cluster while maintaining scalability and reliability.
 
-Kafka Connect is typically used to integrate Kafka with external databases and storage and messaging systems.
-
 The Cluster Operator manages Kafka Connect clusters deployed using the `KafkaConnect` resource and connectors created using the `KafkaConnector` resource.
 
-The following procedures show how to deploy Kafka Connect and set up connectors for streaming data:
+In order to use Kafka Connect, you need to do the following.
 
-* xref:deploying-kafka-connect-{context}[]
-* xref:con-kafka-connect-multiple-instances-{context}[]
-* xref:using-kafka-connect-with-plug-ins-{context}[]
-* xref:assembly-creating-managing-connectors-{context}[]
-* xref:proc-deploying-kafkaconnector-{context}[]
+* xref:deploying-kafka-connect-{context}[Deploy a Kafka Connect cluster]
+* xref:using-kafka-connect-with-plug-ins-{context}[Add connectors to integrate with other systems]
```

Comment:
This list seems strange to me. There are two steps to using Kafka Connect, first you deploy a Connect cluster with the connector plugins you want already installed. Second you start connectors. The second bullet here doesn't make it clear whether it is referring to just starting connectors, or also adding them to the connect image. I couldn't work out what this is linking to, perhaps it would make sense if I could see that, but I wonder if "Start connectors that integrate with.." would make more sense than using the word "Add"?

Reply:
The second bullet groups all the connectors procedures: the two ways you add the plugins by creating a new image. And also the two procedures to manage connectors using the `KafkaConnector` resource and API.  I'm not sure _start_ works in that context.  Here's the section. I thought maybe _introducing connectors_ as an alternative:

![image](https://user-images.githubusercontent.com/47596553/212737790-a175e010-4257-45ce-b3f7-ab34384341c0.png)

Reply:
Maybe _running connectors_ might work here.

Reply:
I guess the bit I was struggling with is that really it makes sense to add connector plugins at the same time as deploying Connect for the first time. Otherwise you deploy Connect, then have to update it to have the plugins you need, then start the connectors. But perhaps it's simpler for the reader if we talk about connectors all in one go, even if it means they are having to re-deploy their Connect workers. I think seeing the context actually using `Adding Connectors` here is ok. However, I think wherever we just refer to starting connectors e.g. with the KafkaConnector resource, we should stay away from the word add.

So I guess I'm proposing:
 - Use the word "Add" when refering to either adding plugins to Connect, or both adding plugins and then starting/managing them
 - Use the word "Run" or "Start" or "Manage" when only refering to starting and managing connectors using e.g. the API or KafkaConnector CR

Does that make sense?

Reply:
Thanks @katheris -- Makes sens. I've gone thgrough the _Adding connectors_ section and I think we're good.

Reply:
Thanks